src/calendar/day/basic/style.js

```diff
@@ -11,6 +11,11 @@ export default function styleConstructor(theme={}) {
       height: 32,
       alignItems: 'center'
     },
+    weekNumberContainer: {
+      width: 32,
+      height: 32,
+      alignItems: 'center'
+    },
```

Comment:
I think you shouldn't duplicate the styling. we can do:
```
 base: {
      width: 32,
      height: 32,
      alignItems: 'center'
 },
 weekNumberContainer: {},
```

You still can override like this:
```
'stylesheet.day.basic': {
    weekNumberContainer: {
      // whatever you want
    }
  }
```

Can you please try it?